frontend/guardian.rb

```diff
@@ -29,31 +28,24 @@ def restart(command)
   return tag_reporter
 end
 
-def close_door(gpio_command)
-  `#{gpio_command} -g write 9 1`
-end
-
-def open_door(gpio_command)
-  `#{gpio_command} -g write 9 0`
+def close_door(gpio_command, door)
+  `#{gpio_command} -g write #{DOOR_PINS[door]} 1`
 end
 
-def accepted_beep(gpio_command)
-  `#{gpio_command} -g write 11 0`
-  sleep 0.1
-  `#{gpio_command} -g write 11 1`
-end
-
-def rejected_beep(gpio_command)
-  accepted_beep(gpio_command)
-  sleep 0.2
-  accepted_beep(gpio_command)
+def open_door(gpio_command, door)
+  `#{gpio_command} -g write #{DOOR_PINS[door]} 0`
 end
 
 $stdout.sync = true
 
 reader_command = '../reader/report_tag'
 gpio_command = 'gpio'
 
+DOOR_PINS = {
```

Comment:
Style suggestion: Place definition of this before the first use on line 35.  Perhaps near the top of the file.